janet/test/gilded-rose.janet

```diff
@@ -0,0 +1,10 @@
+(import ../gilded-rose/main)
+(use judge)
+
+(do
+  (def items 
+    [@{ :name "foo" :quality 3 :sell-in 5}
```

Comment:
This doesn't quite match the starting unit test of the other languages. Could you change the array to only contain "foo" and give it quality 0 and sell-in 0 ?

Reply:
Sorry! I have overlooked that all the other implementations had exactly the same unit test. I've altered the janet test to look more like everyone else's.